src/mocks/handlers/users.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpResponse } from 'msw'
+import { http } from '@/mocks/http'
+import type { components } from '@/api/schema'
+
+type UserResponse = components['schemas']['UserResponse']
+
+const myID = import.meta.env.MY_TRAQ_ID || 'kitsne'
+
+export const usersHandlers = [
+  http.get('/api/me', async () => {
+    return HttpResponse.json({ id: myID, isStaff: true } as UserResponse, { status: 200 })
```

Comment:
openapi-mswの型推論が効くので、ここの型アサーションは不要です
```suggestion
    return HttpResponse.json({ id: myID, isStaff: true }, { status: 200 })
```